Tables.go

```diff
@@ -0,0 +1,469 @@
+//   - outerSize.y < 0.0f  ->  bottom align
+//   - outerSize.y = 0.0f  ->  bottom align, consistent with BeginChild(). not recommended unless table is last item in parent window.
+//   - outerSize.y > 0.0f  ->  fixed child height. recommended when using Scrolling on any axis.
+//   Y without scrolling, we output table directly in parent window:
+//   - outerSize.y < 0.0f  ->  bottom align (will auto extend, unless NoHostExtendV is set)
+//   - outerSize.y = 0.0f  ->  zero minimum height (will auto extend, unless NoHostExtendV is set)
+//   - outerSize.y > 0.0f  ->  minimum height (will auto extend, unless NoHostExtendV is set)
+// About 'innerWidth':
+//   With ScrollX:
+//   - innerWidth  < 0.0f  ->  *illegal* fit in known width (right align from outerSize.x) <-- weird
+//   - innerWidth  = 0.0f  ->  fit in outer_width: Fixed size columns will take space they need (if avail, otherwise shrink down), Stretch columns becomes Fixed columns.
+//   - innerWidth  > 0.0f  ->  override scrolling width, generally to be larger than outerSize.x. Fixed column take space they need (if avail, otherwise shrink down), Stretch columns share remaining space!
+//   Without ScrollX:
+//   - innerWidth          ->  *ignored*
+// Details:
+// - If you want to use Stretch columns with ScrollX, you generally need to specify 'innerWidth' otherwise the concept
+//   of "available space" doesn't make sense.
+// - Even if not really useful, we allow 'innerWidth < outerSize.x' for consistency and to facilitate understanding
+//   of what the value does.
+func BeginTableV(id string, columnsCount int, flags int, outerSize Vec2, innerWidth float32) bool {
+	idArg, idFin := wrapString(id)
+	defer idFin()
+	outerSizeArg, _ := outerSize.wrapped()
+	return C.iggBeginTable(idArg, C.int(columnsCount), C.int(flags), outerSizeArg, C.float(innerWidth)) != 0
+}
+
+// BeginTable calls BeginTableV(id, columnsCount, 0, imgui.Vec2{}, 0.0).
+func BeginTable(id string, columnsCount int) bool {
+	return BeginTableV(id, columnsCount, 0, Vec2{}, 0.0)
+}
+
+// EndTable closes the scope for the previously opened table.
+// only call EndTable() if BeginTable() returns true!
+func EndTable() {
+	C.iggEndTable()
+}
+
+// TableNextRowV appends into the first cell of a new row.
+func TableNextRowV(flags int, minRowHeight float32) {
+	C.iggTableNextRow(C.int(flags), C.float(minRowHeight))
+}
+
+// TableNextRow calls TableNextRowV(0, 0.0).
+func TableNextRow() {
+	TableNextRowV(0, 0.0)
+}
+
+// TableNextColumn appends into the next column (or first column of next row if currently in last column)
+// Return true when column is visible.
+func TableNextColumn() bool {
+	return C.iggTableNextColumn() != 0
+}
+
+// TableSetColumnIndex appends into the specified column. Return true when column is visible.
+func TableSetColumnIndex(columnN int) bool {
+	return C.iggTableSetColumnIndex(C.int(columnN)) != 0
+}
+
+// Tables: Headers & Columns declaration
```

Comment:
I don't think such free-floating comment blocks are included in the generated documentation.
Perhaps add these blocks to a prominent "first" function for the respective functionality.